tasks.py

```diff
@@ -591,7 +591,7 @@ def yamllint(context):
 @task
 def check_migrations(context):
     """Check for missing migrations."""
-    command = "nautobot-server --config=nautobot/core/tests/nautobot_config.py makemigrations --dry-run --check"
+    command = "nautobot-server makemigrations --dry-run --check"
```

Comment:
Was this specific to this app or is it something that needs to be fixed more generally?

Reply:
This was pre-drift manager. This is fixed on drift-manager, but likely copy/pasted from another repo. I found this in Welcome Wizard and I will be checking other repos.

Reply:
It appears to be a problem in the ltm branch of multiple apps, perhaps all of them

Reply:
We're not drift managing ltm so this may have to be manually fixed on every app as we start fixing upstream testing

Reply:
Ack - I missed this was an LTM fix.